libs/core/langchain_core/runnables/base.py

```diff
@@ -1448,6 +1465,29 @@ def mul_two(x: int) -> int:
     last: Runnable[Any, Output]
     """The last runnable in the sequence."""
 
+    def __init__(self, *steps: RunnableLike, name: Optional[str] = None) -> None:
```

Comment:
should we still accept **kwargs and parse them for backwards compatibility

Reply:
Done